packages/client/src/index.ts

```diff
@@ -39,6 +42,63 @@ export class Client extends AutoGeneratedClient {
     super(undefined, clientConfig.apiUrl, axiosClient)
 
     this.config = clientConfig
+    this.axiosClient = axiosClient
+  }
+
+  /**
+   * Creates and uploads a new file in a single step. Returns an object containing the file metadata and the URL to retrieve the file.
+   */
+  public createAndUploadFile = async ({
+    name,
+    data,
+    index,
+    tags,
+    contentType,
+    contentLength,
+    accessPolicies,
+  }: CreateFileProps & { data: Buffer; contentLength?: number }): Promise<GetFileResponse> => {
+    const headers = {
+      ...this.config.headers,
+      'content-length': contentLength,
+    }
+
+    const requestBody: CreateFileBody = {
+      name,
+      tags,
+      index,
+      accessPolicies,
+      contentType,
+    }
+
+    const response = await this.axiosClient
```

Comment:
Why not use `createFile` directly, instead of axios?

Reply:
I tried, but we cannot pass the `contentLength` param to the client method.